docs/concatenating.md

```diff
@@ -0,0 +1,102 @@
+# `StreamingDataFrame.concat()`: concatenating multiple topics into a stream 
+
+Use `StreamingDataFrame.concat()` to combine two or more topics into a new stream containing all the elements from all the topics.
+
+Use it when you need:
+
+- To process multiple topics as a single stream.
+- To combine the branches of the same StreamingDataFrame back together.
+
+## Examples
+
+**Example 1:**  Aggregate e-commerce orders from different locations into one stream and calculate the number of orders made in 1h windows.
+
+```python
+from datetime import timedelta
+
+from quixstreams import Application
+from quixstreams.dataframe.windows import Count
+
+app = Application(...)
+
+# Define the orders topics 
+topic_uk = app.topic("orders-uk")
+topic_de = app.topic("orders-de")
+
+# Create StreamingDataFrames for each location
+orders_uk = app.dataframe(topic_uk)
+orders_de = app.dataframe(topic_de)
+
+# You can also process each topic independently here:
+orders_uk = orders_uk[orders_uk["status"] != "FAILED"]
+orders_de = orders_de[orders_de["status"] != "FAILED"]
```

Comment:
Let's replace it with an operation different from the one for the UK orders to demonstrate that, before the concat, these are distinct pipelines for a reason.